haskell-lsp-types/src/Language/Haskell/LSP/TH/Color.hs

```diff
@@ -0,0 +1,191 @@
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE TemplateHaskell       #-}
+module Language.Haskell.LSP.TH.Color where
+
+import           Data.Aeson.TH
+import           Data.Text (Text)
+import           Language.Haskell.LSP.TH.Constants
+import           Language.Haskell.LSP.TH.List
+import           Language.Haskell.LSP.TH.Location
+import           Language.Haskell.LSP.TH.Message
+import           Language.Haskell.LSP.TH.TextDocument
+import           Language.Haskell.LSP.TH.WorkspaceEdit
+
+{-
+Document Color Request (:leftwards_arrow_with_hook:)
+Since version 3.6.0
+
+The document color request is sent from the client to the server to list all color references found in a given text document. Along with the range, a color value in RGB is returned.
```

Comment:
Can you wrap the long lines, so they can be seen clearly on github? Column 115. Or even 80.